common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -188,6 +189,18 @@ public abstract AcquireTokenResult acquireDeviceCodeFlowToken(final Authorizatio
                 scopes.addAll(interactiveTokenCommandParameters.getExtraScopesToConsent());
             }
 
+            final HashMap<String, String> completeRequestHeaders = new HashMap<>();
+
+            if (interactiveTokenCommandParameters.getRequestHeaders() != null) {
+                completeRequestHeaders.putAll(interactiveTokenCommandParameters.getRequestHeaders());
```

Comment:
This is super-trivial, but we can save some allocations on the map resize if we size it to start with or use the Map/Map constructor to handle this trivially.

Reply:
Yeah, I'll keep as is for now to avoid branched logic for creating hashmap. because this could be null